rust/release/src/main.rs

```diff
@@ -300,6 +304,28 @@ Overview:"#
                 println!("-------------------------\n");
                 println!("Branch to create {}", branch_name);
             } else {
+                // This pathing abuses the fact that we control where this is run from: CI
+                // and in CI this is *usually* run in the rust/release path
+                let ax_cargo = PathBuf::from("../actyx/ax/Cargo.toml").canonicalize()?;
+                let ax_core_cargo = PathBuf::from("../actyx/ax-core/Cargo.toml").canonicalize()?;
+                for (product, v) in new_versions {
+                    let new_version = v.new_version.unwrap();
+                    match product {
+                        Product::Ax => {
+                            eprint!("0.1) Writing new version to \"{}\" ... ", ax_cargo.display());
+                            update_package_version(&ax_cargo, &new_version)?;
+                            repo.add_file(&ax_cargo)?;
+                        }
+                        Product::AxCore => {
+                            eprint!("0.2) Writing new version to \"{}\" ... ", ax_core_cargo.display());
+                            update_package_version(&ax_core_cargo, &new_version)?;
+                            repo.add_file(&ax_core_cargo)?;
+                        }
+                        // We're not updating TOMLs for anything else
```

Comment:
What about updating DATABANK_VERSION? If they aren’t updated at the same time, the build will fail.

Here I suggest again putting that constant into a single-line Rust file so that it can trivially be overwritten.